### PR TITLE
Update free-courses-bn.md

### PR DESCRIPTION
**Pull request by AryavT**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
added python tutorial in Bengali

### Why is this valuable (or not)?
this is going to teach people fluent in Bengali in India to be able to compete with others that have a language advantage
### How do we know it's really free?
because it is a YouTube video
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -274,6 +274,7 @@ Online School
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - STUDY MART
```
Comment (sohansai) [inline courses/free-courses-bn.md:277]: For YouTube, we prefer "playlist" links, not individual video links. 

The link you provided seems different from the title. 

Are you referring to this link: https://www.youtube.com/playlist?list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ    ? 

Additionally, use the title from the resource and replace "|" with "-".

Comment (AryavT) [reply courses/free-courses-bn.md:276]: fixed please approve it is in order now

**Review by AryavT — COMMENTED** [review]
its giving a lint error even though i fixed it, can you help?

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-mr.md [hunk 2]
```diff
@@ -28,6 +28,7 @@
 ### Arduino
 
 * [Arduino Programming in Marathi](https://youtube.com/playlist?list=PL8yOrZ6_TOt5Y-ZG34wLePPZsDaZpatqD) - Asawari Shiposkar
+* [Arduino Programming Timer Switch in Marathi](https://www.youtube.com/watch?v=616TlPXv3-w&list=PLYearvGpQL11nJ0duF_ZTE6Ks4-ML6WBS) - Marathi Computer
```
Comment (eshellman) [inline courses/free-courses-mr.md:31]:
```suggestion
* [Arduino Advanced मराठीमध्ये](https://www.youtube.com/playlist?list=PLYearvGpQL11nJ0duF_ZTE6Ks4-ML6WBS) - Marathi Computer
```

**Pull request merged**